firstvoices/backend/serializers/site_serializers.py

```diff
@@ -69,6 +70,8 @@ class SiteDetailSerializer(SiteSummarySerializer):
     banner_image = ImageSerializer()
     banner_video = VideoSerializer()
 
+    homepage = SiteWidgetListSerializer()
```

Comment:
Another option here would be to set `source="homepage.widgets", many=True` and not have to rearrange things in the serializer